### PR TITLE
policy(CoC): Add code of conduct

### PR DESCRIPTION
**Pull request by Gusted**
@vednoc This is a slightly modified template of the [contributer conviant v2](https://www.contributor-covenant.org/version/2/0/code_of_conduct/) to clarify that this code of conduct's scope is not intended for that USw site.

### CODE REVIEW

**Comment by vednoc** [comment]
No. I'm not approving this PR.

**Comment by vednoc** [comment]
That said, I think [GNU Kind Communications Guidelines][1] would be a much better fit. Like I told Gusted in direct messages: I'm not against the idea, I'm against the way its guidelines are enforced.

[1]: https://www.gnu.org/philosophy/kind-communication.html

**Comment by Gitleptune** [comment]
I think that permanent ban is a bit too harsh unless someone is being a huge knob, and I'm not very keen on the last section that is: 

> Other conduct which could reasonably be considered inappropriate in a professional setting

Because that would: 

1. Cause a lot of troubles for people who don't read *Code of Conducts* (which are a lot) who just want to add a bit of fun to their README.md or are just more lax.
2. Make the community less fun as it now is quite laid back and pleasant to be in because it's not 100% serious.

Otherwise, I think that it is fine otherwise but a bit too wordy for most people to bother reading. You would want people to read it but most wouldn't read 130-ish lines before commenting, opening an issue or something like that.

**Comment by vednoc** [comment]
> You would want people to read it but most wouldn't read 130-ish lines before commenting, opening an issue or something like that.

While I agree with you, I find the GNU one a lot more easy to read. The main reason for that is its tone. I had to not just read, but also dissect Contributor Covenant in order to understand it, and even now I'm not 100% certain I _fully_ understand it.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+- Please don't argue unceasingly for your preferred course of action when a decision for some other course has already been made. That tends to block the activity's progress.
+    
+- If others have irritated you, perhaps by disregarding these guidelines, please don't excoriate them, and especially please don't hold a grudge against them. The constructive approach is to encourage and help other people to do better. When they are trying to learn to do better, please give them plenty of chances.
+
+- If other participants complain about the way you express your ideas, please make an effort to cater to them. You can find ways to express the same points while making others more comfortable. You are more likely to persuade others if you don't arouse ire about secondary things.
+
+- Please don't raise unrelated political issues in USw-related discussions, because they are off-topic.
+
+- Don't be a dick.
```
Comment (vednoc) [inline CODE_OF_CONDUCT.md:47]: Why is this here?

Comment (Gusted) [reply CODE_OF_CONDUCT.md:47]: Request from @narcolepticinsomniac

Comment (vednoc) [reply CODE_OF_CONDUCT.md:47]: Was it an actual request or just a comment?

Comment (Gusted) [reply CODE_OF_CONDUCT.md:47]: I agree, good guideline. Consider it as an TL;DR

Comment (vednoc) [reply CODE_OF_CONDUCT.md:104]: I can't consider it as Tl;DR and neither can new contributors that are meant to read this document. As it stands, it doesn't belong there.

Comment (narcolepticinsomniac) [reply CODE_OF_CONDUCT.md:104]: > Was it an actual request or just a comment?

Offhanded remark. If anyone's taking this seriously, I promise you, I could not care less.

Comment (vednoc) [reply CODE_OF_CONDUCT.md]: Just as I thought. I'll let this PR stay open until tomorrow and that's when we'll decide what to do.
